resources/language/resource.language.hu_hu/strings.po

```diff
@@ -0,0 +1,977 @@
+
+msgctxt "#30207"
+msgid "Set this device as main auto-updates manager"
+msgstr "Eszköz beállítása automatikus frissítéskezelőként"
+
+msgctxt "#30208"
+msgid "Check if this device is the main auto-update manager"
+msgstr "Megnézi hogy ez az eszköz, automatikus frissítéskezelő-e"
+
+msgctxt "#30209"
+msgid "This device now handles the auto-updates of shared-library"
+msgstr "Eszköz most automatikus frissítéskezelőként beállítva"
+
+msgctxt "#30210"
+msgid "This device handles the auto-updates of shared-library"
+msgstr "Eszköz beállítása automatikus frissítéskezelőként"
+
+msgctxt "#30211"
+msgid "Another device handles the auto-updates of shared-library"
+msgstr "Másik eszköz beállítása automatikus frissítéskezelőként"
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of shared-library"
+msgstr "Nincs eszköz beállítva automatikus frissítéskezelőként"
+
+msgctxt "#30213"
+msgid "InputStream Helper settings..."
+msgstr "InputStream Helper beállítások..."
+
+msgctxt "#30214"
+msgid "Force the update of my list"
+msgstr "Saját listám kényszerített frissítése"
+
+msgctxt "#30215"
+msgid "Highlight titles included in my list with a color"
+msgstr "Színkiemelés a saját listám elemein"
+
+msgctxt "#30216"
+msgid "Show ESN in use"
+msgstr "ESN használat mutatása"
+
+msgctxt "#30217"
+msgid "Reset ESN"
+msgstr "ESN reset"
+
+msgctxt "#30218"
+msgid "Are you sure you want to reset the ESN?\r\nIf you confirm, wait until you receive the login message."
+msgstr "Biztosan vissza akarja állítani az ESN-t?\r\nHa megerősíti, várjon, amíg megkapja a bejelentkezési üzenetet."
+
+msgctxt "#30219"
+msgid "Disable notification for synchronization completed"
+msgstr "A Szinkronizálás kész, infó elrejtése"
+
+msgctxt "#30220"
+msgid "The Kodi library has been updated"
+msgstr "Kodi médiatár frissítve"
+
+msgctxt "#30221"
+msgid "Owner account"
+msgstr "Felhasználói fiók "
+
+msgctxt "#30222"
+msgid "Kid account"
+msgstr "Gyermek fiók"
+
+msgctxt "#30223"
+msgid "Synchronization now uses \"My List\" of profile {}"
+msgstr "A szinkronizáció most a  \"Saját listámat\" használja a profilhoz {}"
+
+msgctxt "#30224"
+msgid "Auto update mode"
+msgstr "Automata frissítési mód"
+
+msgctxt "#30225"
+msgid "Manual"
+msgstr "Kézi"
+
+msgctxt "#30226"
+msgid "Scheduled"
+msgstr "Ütemezett"
+
+msgctxt "#30227"
+msgid "Synchronize Kodi library with My List"
+msgstr "Kodi médiatár szinronizálása a saját listámmal"
+
+msgctxt "#30228"
+msgid "Use \"My list\" of the current profile"
+msgstr "Használd a \"Saját listámat\" a jelenlegi profilhoz"
+
+msgctxt "#30229"
+msgid "Show the name of the selected profile"
+msgstr "Kiválasztott profilnév mutatása"
+
+msgctxt "#30230"
+msgid "Check for updates now"
+msgstr "Frissítés keresése most"
+
+msgctxt "#30231"
+msgid "Do you want to check update now?"
+msgstr "Akarsz most keresni frissítést?"
+
+msgctxt "#30232"
+msgid "Restrict by Maturity Level"
+msgstr "Kor szeint korlátozva"
+
+msgctxt "#30233"
```

Comment:
from this line to the end of file, all is changed, compare with en file language